resalloc-ibm-cloud.spec

```diff
@@ -0,0 +1,55 @@
+%global desc %{expand:
+Helper scripts for the Resalloc server (mostly used by Copr build system)
+for maintaining VMs in IBM Cloud (starting, stopping, cleaning orphans, etc.).
+}
+
+Name:           resalloc-ibm-cloud
+Version:        0.1.0
+Release:        1%{?dist}
+Summary:        Resource allocator scripts for IBM cloud
+
+License:        GPLv2+
+URL:            https://github.com/fedora-copr/%{name}
+Source0:        %{url}/archive/refs/tags/%{name}-%{version}.tar.gz
+
+
+BuildArch:      noarch
+
+BuildRequires:  python3-devel
+BuildRequires:  python3-ibm-vpc
+BuildRequires:  python3-ibm-cloud-sdk-core
+BuildRequires:  pyproject-rpm-macros
+
+
+%description
+%{desc}
```

Comment:
description should end with a dot
https://docs.fedoraproject.org/en-US/packaging-guidelines/#_summary_and_description